src/beast/beast/module/core/native/linux_Files.cpp

```diff
@@ -145,9 +144,9 @@ File File::getSpecialLocation (const SpecialLocationType type)
 class DirectoryIterator::NativeIterator::Pimpl
 {
 public:
-    Pimpl (const File& directory, const String& wildCard_)
+    Pimpl (const File& directory, const String& _wildCard)
```

Comment:
Jeepers!  I know that the standard only reserves names with leading underscores if they are followed by an upper case letter of another underscore (http://stackoverflow.com/questions/228783/what-are-the-rules-about-using-an-underscore-in-a-c-identifier).

However my habit for years has been to never name anything with a leading underscore.  Sutter and Alexandrescu agree with my take in **C++ Coding Standards** *Item 0*: "a) never use 'underhanded names,' ones that begin with an underscore or contain a double underscore;...".  So with this change I just went from uncomfortable to more uncomfortable.  But I can't say that it violates the standard.